### PR TITLE
Typo improvement: Week 02 - Python language fundamentals 

### PR DESCRIPTION
**Pull request by LennartKoe**

### CODE REVIEW

File: book/week_02/01-Language-Fundamentals.ipynb [hunk 1]
```diff
@@ -1042,7 +1042,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "It is immutability which makes tuples useful, but for beginners this is not really obvious at the first sight. We will get back to tuples later in the lecture."
+    "It is immutable which makes tuples useful, but for beginners this is not really obvious at the first sight. We will get back to tuples later in the lecture."
```
Comment (fmaussion) [inline book/week_02/01-Language-Fundamentals.ipynb:1045]: I disagree on this one, but will revert the change locally

**Comment by fmaussion** [comment]
Thanks a lot!

File: book/week_02/01-Language-Fundamentals.ipynb (hunk 1 above)
Comment (LennartKoe) [reply book/week_02/01-Language-Fundamentals.ipynb:1045]: Oh well, reads way better how it is now!
